Packs/Core/Playbooks/playbook-T1059_-_Command_and_Scripting_Interpreter.yml

```diff
@@ -1152,6 +1169,91 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "55":
+    id: "55"
+    taskid: c16a3995-ec20-430a-84f3-1a71844b1316
+    type: regular
+    task:
+      id: c16a3995-ec20-430a-84f3-1a71844b1316
+      version: -1
+      name: Get CMD prevalence
+      description: Get the prevalence of a process_command_line, identified by process_command_line.
+      script: Cortex Core - IR|||core-get-cmd-analytics-prevalence
+      type: regular
+      iscommand: true
+      brand: Cortex Core - IR
+    nexttasks:
+      '#none#':
+      - "56"
+    scriptarguments:
+      process_command_line:
+        complex:
+          root: alert
+          accessor: targetprocesscmd
```

Comment:
The command can be found in additional fields. Please take the output from the commandline analysis subplaybook